rtaudio_c.cpp

```diff
@@ -126,6 +126,7 @@ int rtaudio_open_stream(rtaudio_t audio,
 			unsigned int *buffer_frames, rtaudio_cb_t cb,
 			void *userdata, rtaudio_stream_options_t *options,
 			rtaudio_error_cb_t errcb) {
+  (void)errcb;
```

Comment:
Better to just put `rtaudio_error_cb_t /*errcb*/` in the parameter list.  If `errcb` is unnamed then gcc doesn't complain.

Reply:
Sorry, I have to argue with this. I don't think it's valid C99 (it does work in C++ only), while casting unused parameters to void is a portable and a rather common solution.

```
$ gcc -std=c99 -Wall -Wextra -pedantic -c x.c
x.c: In function ‘foo’:
x.c:1:1: error: parameter name omitted
 int foo(int) { return 0; }
```

Reply:
Huh. TIL.